### PR TITLE
Revert "Enable resetting of batchnorm running stats and cumulative ("simple") moving average"

### PR DESCRIPTION
**Pull request by soumith**
Reverts pytorch/pytorch#5766

The PR had issues with loading older checkpoints. We'll re-PR the patch once we issue a fix for older checkpoints additionally

### CODE REVIEW

**Comment by soumith** [comment]
cc: @jma127

**Comment by jma127** [comment]
OK, no worries. Thanks for letting me know!
